azure-pipelines/templates/stages/build.yml

```
@@ -79,4 +79,23 @@ stages:
       displayName: Pagkage executables
 
     - publish: $(Build.ArtifactStagingDirectory)/package-$(target)
-      artifact: package-$(target)
\ No newline at end of file
+      artifact: package-$(target)
+
+    - job: PublishScripts
+      dependsOn: Build
+      condition: succeeded()
+      steps:
+        - checkout: self
+        - script: |
```

I'd use either `bash` or `pwsh` instead of `script` when reading this I had to google which one is used for `script` steps and I still don't know since I don't know the vmImage ^^

```suggestion
        - bash: |
```
```suggestion
        - pwsh: |
```